docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>Maybe clearer:</p><p> </p><p>So far, we have looked at examples where we adjusted the color and linestyle properties of the plotting backend using<code>pc_kwags</code> and <code>aes_map</code>.  Next, we will see how to use these arguments to do much more complex adjustments. </p><p> </p><p>When adjusting the aesthetics of a graphical element, the only limitation is that the keys and values are valid passed to a graphical property are valid for the specific backend function being used. As long as this is satisfied, anything goes!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>